src/GithubMergeTool/GithubMergeTool.cs

```diff
@@ -94,11 +94,31 @@ public class GithubMergeTool
     ""sha"": ""{srcSha}""
 }}";
 
+            Console.WriteLine("Creating branch");
             response = await _client.PostAsyncAsJson($"repos/{repoOwner}/{repoName}/git/refs", body);
 
             if (response.StatusCode != HttpStatusCode.Created)
             {
-                return (false, response);
+                // Branch already exists. Hard reset to the new SHA
```

Comment:
❓ What if the branch exists, and has been manually updated to address a merge conflict?

Reply:
➡️ The answer is in the commit message. For this case, there is a branch but no pull request.

Reply:
If the PR is open then we should have bailed out earlier in this method.